.github/ISSUE_TEMPLATE/BUG_REPORT.md

```diff
@@ -32,3 +32,13 @@ Any other relevant information. For example, why do you consider this a bug and
 -   Django version: Look in your requirements.txt, or run `pip show django | grep Version`.
 -   Wagtail version: Look at the bottom of the Settings menu in the Wagtail admin, or run `pip show wagtail | grep Version:`.
 -   Browser version: You can use https://www.whatsmybrowser.org/ to find this out.
+
+### Working on this
```

Comment:
```suggestion
### Contribute to this
```

Semantics I know, but I often try to avoid the term 'work' and keep to contributing.

Reply:
@lb- I'd disagree here - the aim of this section is to hopefully catch the attention of people who would post a comment asking "can I work on this?". That's going to be more effective if we match the language people are already using.

(Also, to me "contributing" to an issue would include things like replicating the issue or suggesting technical approaches to it, which are welcome but not what this particular paragraph is about - we're not asking people to leave a comment before doing those things.)

Reply:
Fair enough, happy to go with this direction then. Thanks for explaining.